Extensions/Physics2Behavior/physics2runtimebehavior.ts

```diff
@@ -241,13 +281,29 @@ namespace gdjs {
     layers: any;
     masks: any;
     shapeScale: number = 1;
+    // Array containing the beginning of contacts reported by onContactBegin. Each contact
+    // should be unique to avoid recording glitches where the object loses and regain
+    // contact between two frames. The array is updated each time the method
+    // onContactBegin is called by the listener, which is only called when stepping
+    // the world i.e. in the first preEvent called by a physics behavior. This array is
+    // cleared just before stepping the world.
```

Comment:
This should be a documentation comment. Even if it's a private member, it allow to read the comment by hovering the attribute in the code.